zcncore/transaction_base.go

```diff
@@ -361,6 +359,14 @@ func (t *Transaction) submitTxn() {
 				return
 			}
 
+			if res.StatusCode != http.StatusOK {
```

Comment:
ah, giving none `StatusOK` means the txn execution failed due to some reason("invalid txn nonce" for example), the txn is actually reached to the miners, and we should return the error to the caller through the `resultC` channel. 

Shouldn't the error returned by`res, err := req.Post()` give the clue about the chaos caused node unreachable info?

Reply:
No, on unreachable nodes status is 400 and req.Post() doesn't return any error.